lib/Driver/Driver.cpp

```diff
@@ -268,6 +268,47 @@ class Driver::InputInfoMap
 };
 using InputInfoMap = Driver::InputInfoMap;
 
+// Get the filename for build record.
+// Additionally set 'outputBuildRecordForModuleOnlyBuild' to true if this is
+// full compilation with swiftmodule.
```

Comment:
Nitpick: This can be a doc comment.

Reply:
Suggest adding a line to say that returns true for error.